### ISSUE TITLE
Necro combat with wounds

### ISSUE THREAD

**Issue by Kaesken** [rpherbig/dr-scripts#5637]
Unfortunately I have to sometimes manually have her eat herbs to heal (long story short, extremely old toon, trained poorly so she doesn't have healing spells yet).  I forgot to check her herb intake and got this during her hunting, figured you guys would like a copy of it.

```
[combat-trainer]>wield my scimitar

Your right hand is too injured to draw the scimitar!
> 
* A sickly blightwater nyad feints low at you.  You partially block with a leather shield reinforced with riveted burgundy leather straps.  
[You're solidly balanced with opponent in good position.]
The blightwater nyad moves into a position to parry.
> 
* A sickly blightwater nyad thrusts a rotting driftwood staff at you.  You barely block with a leather shield reinforced with riveted burgundy leather straps.  
[You're solidly balanced with opponent in superior position.]
* A sickly blightwater nyad sidesteps and bashes at you.  You barely block with a leather shield reinforced with riveted burgundy leather straps.  
[You're solidly balanced with opponent in very strong position.]
You've gained a new rank in your perceptual acuity.
> 
The blightwater nyad moves into a position to parry.
> 
[combat-trainer: *** No match was found after 15 seconds, dumping info]

[combat-trainer: messages seen length: 12]

[combat-trainer: message: The blightwater nyad moves into a position to parry.]

[combat-trainer: message: You've gained a new rank in your perceptual acuity.]

[combat-trainer: message: [You're solidly balanced with opponent in very strong position.]]

[combat-trainer: message: * A sickly blightwater nyad sidesteps and bashes at you.  You barely block with a leather shield reinforced with riveted burgundy leather straps.  ]

[combat-trainer: message: [You're solidly balanced with opponent in superior position.]]

[combat-trainer: message: * A sickly blightwater nyad thrusts a rotting driftwood staff at you.  You barely block with a leather shield reinforced with riveted burgundy leather straps.  ]

[combat-trainer: message: Butcher's Eye  (2 roisaen)]

[combat-trainer: message: Obfuscation  (18 roisaen)]

[combat-trainer: message: The blightwater nyad moves into a position to parry.]

[combat-trainer: message: [You're solidly balanced with opponent in good position.]]

[combat-trainer: message: * A sickly blightwater nyad feints low at you.  You partially block with a leather shield reinforced with riveted burgundy leather straps.  ]

[combat-trainer: message: Your right hand is too injured to draw the scimitar!]

[combat-trainer: checked against [/You draw/i, /You deftly remove/i, /You slip/i, /Wield what/i, /With a flick of your wrist you stealthily unsheathe/i]]

[combat-trainer: for command wield my scimitar]
```

**Comment by MahtraDR** [comment]
@Kaesken what should happen here  Instead of what is happening? Making it hang for 15 seconds is easy, but I feel not the right thing either.

**Comment by Kaesken** [comment]
Well this was when it was trying to swap weapons but she had wounds on her because I don't have her set to go to the NPC Healer.  I've noticed the herbs under the training_info which i've set but I haven't checked if it actually works the way it should just yet.   I attempted to try out the heal-remedy but because Hib uses "solution", I noticed that verb was not set to that script so I'm trying to figure something out for it which i'm not quite sure yet.

What I'd like to happen though...
When this happens, if you're not a Necromancer, kill hunting-buddy and go to an NPC healer or back to your safe-room and trigger eating your herbs to heal up.
